plugins/UM3NetworkPrinting/NetworkClusterPrinterOutputDevice.py

```diff
@@ -111,6 +112,20 @@ def __init__(self, key, address, properties, api_prefix, plugin_path):
         temporary_translation3 = i18n_catalog.i18n("{printer_name} has finished printing '{job_name}'. Please collect the print and confirm clearing the build plate.") #When finished.
         temporary_translation4 = i18n_catalog.i18n("{printer_name} is reserved to print '{job_name}'. Please change the printer's configuration to match the job, for it to start printing.") #When configuration changed.
 
+    ##  No authentication, so requestAuthentication should do exactly nothing
+    @pyqtSlot()
+    def requestAuthentication(self, message_id = None, action_id = "Retry"):
+        Logger.log("d", "requestAuthentication for Cura Connect - nothing to be done")
```

Comment:
No need to log this imho. I'd also like to recommend using the @override decorator.